pom.xml

```diff
@@ -129,6 +129,12 @@
                                     </limits>
                                 </rule>
                             </rules>
+                            <excludes>
+                                <!-- These are reporting false positives that are already being tested elsewhere -->
```

Comment:
Understand the domain & SpringBootApp excludes, but excluding everything under `controller` might be too much: there is code in there that needs coverage checks. Let's find out what's going on.

Also, is there a way to specify the excludes within individual Maven modules?

Reply:
I think I actually just wanted the controller class itself since there's an unimplemented method in there, I'm not sure if there's a way to just exclude individual methods.